lib/css_parser/rule_set.rb

```diff
@@ -326,8 +340,8 @@ def create_shorthand_properties!(properties, shorthand_property) # :nodoc:
       values = []
       properties_to_delete = []
       properties.each do |property|
-        if @declarations.key?(property) and not @declarations[property][:is_important]
-          values << @declarations[property][:value]
+        if @declarations.key?(property) and not @declarations[property].important
```

Comment:
>`next unless @declarations[property]&.important`
I tried to minimally change the code which is not directly affected by the changes, I'll try to take a look on further improvements when the series will be merged.

Reply:
should even be:
```
next unless declaration = @declarations[property]
next unless declaration.important
values << declaration.value
```